oslabs-beta/Kubiquity
# Cpu middleware

WHAT DOES THIS PR DO?

Adds CPU metrics of each pod

HOW TO TEST THIS PR:

Ensure minikube is running with `minikube start`
Run `gh pr checkout 33 && npm install && npm run backend`
Send a GET request to localhost:3000/metrics and ensure you get a console log of an array of objects back with podId and cpu with no cpu values of 0 and podIds of undefined.
